package.json

```diff
@@ -18,7 +18,7 @@
     "prettier-check": "prettier --check \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "test": "pnpm jest",
     "jest": "jest -c jest.config.js --",
-    "jest:e2e": "jest -c jest.config.e2e.js --",
+    "jest:e2e": "jest -c jest.config.e2e.js --forceExit",
```

Comment:
I added --forceExit because after e2e tests are finished, there are a few open Redis connections and open fetch or logger( this I'm not sure). Jest will be hung and not exit.
https://github.com/Seneca-CDOT/telescope/blob/95dde6cb4d852d9967dc312980a3cb1f1305e334/src/api/parser/src/lib/queue.js#L9-L10